### ISSUE TITLE
models weirdly offset and crooked

### ISSUE THREAD

**Issue by jurassicjordan** [kitsune-ONE-team/KITSUNETSUKI-Asset-Tools#7]
depending on the software I load the model into the model will be offset from its origin in both translation and rotation

**Comment by Yonnji** [comment]
Have you tried applying transforms in Blender before exporting?

**Comment by jurassicjordan** [comment]
> Have you tried applying transforms in Blender before exporting?

Yes I have. My only fix is to just rotate the model in the vtubing software

**Comment by Yonnji** [comment]
Can you upload the model to test? You can remove most meshes/vertices if you worry about the model.
